tests/unit/states/test_reg.py

```diff
@@ -48,8 +49,12 @@ def test_present(self):
                                            {'vdata': 'a', 'success': True},
                                            {'vdata': 'a', 'success': True}])
         mock_t = MagicMock(return_value=True)
+        mock_cast = MagicMock(
+            return_value=salt.utils.stringutils.to_unicode(vdata)
+        )
```

Comment:
Could this simply be:
```
mock_cast = MagicMock(return_value=vdata)
```
Because vdata is already unicode (on PY2) due to `from __future__ import unicode_literals`?

Or am I missing something that requires the explicit cast?

Reply:
No, it's my own idiocy. I saw that that's what `cast_vdata` was returning, so I thought I should make the mock the same. The tests pass without the `to_unicode`. I'll remove it.